pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -142,36 +150,24 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
     pybind11 \
     pyyaml \
     # needed by fastai.distributed
-    accelerate \ 
-    cryptography \
-    cmake \
+    accelerate \
     scipy \
     click \
     # patch CVE
     "pillow>=9.4" \
     psutil \
     ipython \
- # pin dgl to 0.9.1 as upstream test has not caught up, currently having accuracy error
- && /opt/conda/bin/mamba install -c dglteam -y dgl \
+ && /opt/conda/bin/mamba install -c dglteam dgl \
```

Comment:
do you need `-y` option here?

Reply:
-y flag makes it quicker for the pipeline I think

Reply:
right, I think you missed adding `-y` option here